style.css

```
@@ -1,164 +1,2362 @@
+    max-height: 450px;
+    background: var(--surface-2) !important;
+    border: 1px solid var(--glass-border);
+    border-radius: 12px;
+    box-shadow: var(--glass-shadow);
+    -webkit-backdrop-filter: blur(var(--glass-blur)) saturate(120%);
+    backdrop-filter: blur(var(--glass-blur)) saturate(120%);
+    z-index: 2147483647 !important; /* above result items */
+    display: none;
+    overflow: hidden;
+    isolation: isolate; /* prevent parent backdrop bleed */
+    display: flex;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

display is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)